Compilation problem with git master on Ubuntu 20.04

```bash
$ rm -rf ./build/ && meson . build && ninja -C build && sudo ninja -C build install
The Meson build system
Version: 0.53.2
Source dir: /home/kelson/code/kiwix-lib
Build dir: /home/kelson/code/kiwix-lib/build
Build type: native build
Project name: kiwix-lib
Project version: 9.1.2
C++ compiler for the host machine: ccache c++ (gcc 9.3.0 "c++ (Ubuntu 9.3.0-10ubuntu2) 9.3.0")
C++ linker for the host machine: c++ ld.bfd 2.34
Host machine cpu family: x86_64
Host machine cpu: x86_64
Run-time dependency threads found: YES
Found pkg-config: /usr/bin/pkg-config (0.29.1)
Run-time dependency icu-i18n found: YES 66.1
Run-time dependency libzim found: YES 6.1.2
Run-time dependency pugixml found: YES 1.10
Run-time dependency libcurl found: YES 7.68.0
Run-time dependency libmicrohttpd found: YES 0.9.69
Has header "mustache.hpp" : YES 
Program kiwix-compile-resources found: YES (/home/kelson/code/kiwix-lib/scripts/kiwix-compile-resources)
Program python3 found: YES (/usr/bin/python3)
Configuring kiwix_config.h using configuration
Run-time dependency GTest found: YES 1.9.0
Configuring kiwix.pc using configuration
Build targets in project: 12

Found ninja-1.8.2 at /home/kelson/bin/ninja
ninja: Entering directory `build'
[10/50] Generating resources with a custom command.
skin/jquery-ui/jquery-ui.structure.min.css ['skin', 'jquery_ui', 'jquery_ui_structure_min_css']
skin/jquery-ui/jquery-ui.min.js ['skin', 'jquery_ui', 'jquery_ui_min_js']
skin/jquery-ui/external/jquery/jquery.js ['skin', 'jquery_ui', 'external', 'jquery', 'jquery_js']
skin/jquery-ui/images/ui-bg_flat_0_aaaaaa_40x100.png ['skin', 'jquery_ui', 'images', 'ui_bg_flat_0_aaaaaa_40x100_png']
skin/jquery-ui/images/ui-bg_flat_75_ffffff_40x100.png ['skin', 'jquery_ui', 'images', 'ui_bg_flat_75_ffffff_40x100_png']
skin/jquery-ui/images/ui-icons_222222_256x240.png ['skin', 'jquery_ui', 'images', 'ui_icons_222222_256x240_png']
skin/jquery-ui/images/ui-bg_glass_55_fbf9ee_1x400.png ['skin', 'jquery_ui', 'images', 'ui_bg_glass_55_fbf9ee_1x400_png']
skin/jquery-ui/images/ui-bg_highlight-soft_75_cccccc_1x100.png ['skin', 'jquery_ui', 'images', 'ui_bg_highlight_soft_75_cccccc_1x100_png']
skin/jquery-ui/images/ui-bg_glass_65_ffffff_1x400.png ['skin', 'jquery_ui', 'images', 'ui_bg_glass_65_ffffff_1x400_png']
skin/jquery-ui/images/ui-icons_2e83ff_256x240.png ['skin', 'jquery_ui', 'images', 'ui_icons_2e83ff_256x240_png']
skin/jquery-ui/images/ui-icons_cd0a0a_256x240.png ['skin', 'jquery_ui', 'images', 'ui_icons_cd0a0a_256x240_png']
skin/jquery-ui/images/ui-icons_888888_256x240.png ['skin', 'jquery_ui', 'images', 'ui_icons_888888_256x240_png']
skin/jquery-ui/images/ui-bg_glass_75_e6e6e6_1x400.png ['skin', 'jquery_ui', 'images', 'ui_bg_glass_75_e6e6e6_1x400_png']
skin/jquery-ui/images/animated-overlay.gif ['skin', 'jquery_ui', 'images', 'animated_overlay_gif']
skin/jquery-ui/images/ui-bg_glass_75_dadada_1x400.png ['skin', 'jquery_ui', 'images', 'ui_bg_glass_75_dadada_1x400_png']
skin/jquery-ui/images/ui-icons_454545_256x240.png ['skin', 'jquery_ui', 'images', 'ui_icons_454545_256x240_png']
skin/jquery-ui/images/ui-bg_glass_95_fef1ec_1x400.png ['skin', 'jquery_ui', 'images', 'ui_bg_glass_95_fef1ec_1x400_png']
skin/jquery-ui/jquery-ui.theme.min.css ['skin', 'jquery_ui', 'jquery_ui_theme_min_css']
skin/jquery-ui/jquery-ui.min.css ['skin', 'jquery_ui', 'jquery_ui_min_css']
skin/caret.png ['skin', 'caret_png']
skin/taskbar.js ['skin', 'taskbar_js']
skin/taskbar.css ['skin', 'taskbar_css']
skin/block_external.js ['skin', 'block_external_js']
templates/search_result.html ['templates', 'search_result_html']
templates/no_search_result.html ['templates', 'no_search_result_html']
templates/404.html ['templates', '_404_html']
templates/500.html ['templates', '_500_html']
templates/index.html ['templates', 'index_html']
templates/suggestion.json ['templates', 'suggestion_json']
templates/head_part.html ['templates', 'head_part_html']
templates/taskbar_part.html ['templates', 'taskbar_part_html']
templates/external_blocker_part.html ['templates', 'external_blocker_part_html']
templates/captured_external.html ['templates', 'captured_external_html']
opensearchdescription.xml ['opensearchdescription_xml']
[49/50] Compiling C++ object 'test/9f86d08@@server@exe/server.cpp.o'.
FAILED: test/9f86d08@@server@exe/server.cpp.o 
ccache c++ -Itest/9f86d08@@server@exe -I/usr/local/include -I/usr/include/x86_64-linux-gnu -fdiagnostics-color=always -pipe -D_FILE_OFFSET_BITS=64 -Wall -Winvalid-pch -Wnon-virtual-dtor -Werror -std=c++11 -g -pthread -DGTEST_HAS_PTHREAD=1 -MD -MQ 'test/9f86d08@@server@exe/server.cpp.o' -MF 'test/9f86d08@@server@exe/server.cpp.o.d' -o 'test/9f86d08@@server@exe/server.cpp.o' -c ../test/server.cpp
../test/server.cpp: In member function ‘virtual void ServerTest_ETagHeaderIsSetAsNeeded_Test::TestBody()’:
../test/server.cpp:283:8: error: suggest explicit braces to avoid ambiguous ‘else’ [-Werror=dangling-else]
  283 |     if ( res.etag_expected )
      |        ^
cc1plus: all warnings being treated as errors
ninja: build stopped: subcommand failed.
```

--- a/test/server.cpp
+++ b/test/server.cpp
@@ -280,8 +280,9 @@ TEST_F(ServerTest, ETagHeaderIsSetAsNeeded)
   for ( const Resource& res : all200Resources() ) {
     const auto responseToGet = zfs1_->GET(res.url);
     EXPECT_EQ(res.etag_expected, responseToGet->has_header("ETag")) << res;
-    if ( res.etag_expected )
+    if ( res.etag_expected ) {
       EXPECT_TRUE(is_valid_etag(responseToGet->get_header_value("ETag")));
+    }
   }
 }
 

